modules/product/src/main/java/com/opengamma/strata/product/fx/FxVanillaOption.java

```diff
@@ -46,26 +40,17 @@
  * An FX option is a financial instrument that provides an option based on the future value of
  * a foreign exchange. The option is European, exercised only on the exercise date.
  * <p>
- * If the option is a call, the option holder has the right to enter into the specified exchange.
- * If the option is a put, the option holder has the right to enter into the opposite of the specified exchange.
- * <p>
  * For example, a call on a 'EUR 1.00 / USD -1.41' exchange is the option to
  * perform a foreign exchange on the expiry date, where USD 1.41 is paid to receive EUR 1.00.
- * A put on the same exchange is the option to pay EUR 1.00 and receive USD 1.41.
+ * <p>
+ * Note that all of the computation in product pricers must be based on the counter currency of the underlying FX 
+ * transaction. Thus price, pv and risk measures of the product will be expressed in USD by default for the above example. 
```

Comment:
Comments about the pricer should be at the pricer level, not in the instrument description.